pages/home.md

```diff
@@ -67,7 +63,7 @@ permalink: /
           <div class="column is-offset-1 is-5">
             <h2 class="title">Store anything</h2>
             <p class="content is-large">
-              Any kind of data can be stored in a Solid pod, including regular files that you might store in a Google Drive or Dropbox folder, but it is the ability to store <b>Linked Data</b> that makes Solid special.
+              Any kind of data can be stored in a Solid pod, including regular files that you might store in a Google Drive or Dropbox folder. But it is the ability to store <b>Linked Data</b> that makes Solid special.
```

Comment:
```suggestion
              Any kind of data can be stored in a Solid pod, including regular files that you might store in a Google Drive or Dropbox folder. ~The ability to store <b>Linked Data</b> is what makes Solid special.
```

Reply:
Q:  is the tilde ~ in front of "The" just a typo?

Reply:
typo; fixed

Reply:
Thank you!  Do you think that second sentence might work better if flipped? as in "... Google Drive or Dropbox folder.  What makes Solid special is the ability to store <strong>Linked Data</strong>."